src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -44,9 +46,12 @@ public function configureOptions(OptionsResolver $resolver)
                     return null;
                 }
 
+                $this->locale = $options['locale'];
```

Comment:
Even if the use case looks weird, this may be overridden for each use of the type in the same request.

Reply:
@Koc: Do you have some time to finish this?

Reply:
yes, will try do it during this last week before stabilization phase.